control/trajectory_follower_nodes/src/lateral_controller_node.cpp

```diff
@@ -181,7 +181,6 @@ LateralController::~LateralController()
 void LateralController::onTimer()
 {
   if (!checkData() || !updateCurrentPose()) {
-    publishCtrlCmd(getStopControlCommand());
```

Comment:
@TakaHoribe Is it ok not to publish stop control command when it doesn't receive the data?
If we apply this change, it doesn't publish any control command when this situation.

Reply:
I made the change based on a discussion in Slack. If this change is not needed, it can be reverted.

Reply:
Publishing stop control command in this layer isn't good because it cannot be taken into account in the fail-safe system.
So I believe it should stop publishing commands and let the fail-safe system know the error.

Reply:
OK. I agree your comment. Thank you!
(I just worried about changing behavior with this change.)

Reply:
@rej55 @kenji-miyake Could you write here how the system behavior will change by this fix in current products? Just for in case.
Since JpnTaxi doesn't see some diags, it would be a bad situation.

Reply:
The stop command from MPC is overwritten by the velocity control, so it has no effect. Excuse me.